crucible/from-file-json-manipulation-api/run.sh

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
+
+# user configurable options #####################################################
+
+target="centosstream9-rt"
+
+userenv="default"
+
+#osruntime="chroot"
+osruntime="podman"
+stress_ng_osruntime="chroot"
+
+tools="no"
+#tools="yes"
+
+#stress_ng="no"
+stress_ng="yes"
+
+samples=3
+
+mode="run"
+#mode="test"
+
+from_file="oslat.json"
```

Comment:
Perhaps these settings could be in a separate file under run-file-api/config and run.sh just source the config file... my point is to keep the run.sh a completely independent and reusable script from the data itself.